rollup.config.js

```diff
@@ -19,7 +20,17 @@ const config = {
   ]
 }
 
-if (process.env.BABEL_ENV === 'production') {
+if (process.env.STRICT !== 'true') {
+  config.plugins.push(
+    strip({
+      debugger: true,
+      functions: ['console.*'],
+      sourceMap: true
+    })
+  )
+}
+
+if (process.env.MINIFY) {
```

Comment:
Had to update the rollup config here. I don't want warnings to hit the main build. Strict mode only